src/ui/windows/TogglDesktop/TogglDesktop/ui/ViewModels/TimelineViewModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Reactive;
+using System.Threading.Tasks;
+using ReactiveUI;
+using ReactiveUI.Fody.Helpers;
+
+namespace TogglDesktop.ViewModels
+{
+    public class TimelineViewModel : ReactiveObject
+    {
+        public TimelineViewModel()
+        {
+            this.WhenAnyValue(x => x.RecordActivity).Subscribe(x => SaveRecordActivity());

```

Comment:
There is a nicer way to offload a subscription to the TaskPool:
```suggestion
            this.WhenAnyValue(x => x.RecordActivity).ObserveOn(RxApp.TaskpoolScheduler).Subscribe(Toggl.SetTimelineRecordingEnabled);

```
and no need for `SaveRecordActivity()`